technology-considerations/API_management_AppSync_GraphQL_REST.md

```diff
@@ -0,0 +1,62 @@
+# Options for API management
+
+We want to have an intermediate component between our frontend (which consists of the browser and the Play web application)
+and the database. 
+This component will represent our APIs. The APIs will include virus scanning, format checking, validations of the format.
+
+The APIs can use the [REST architectural style]  (https://en.wikipedia.org/wiki/Representational_state_transfer),
```

Comment:
The Markdown for this link isn't working - I think it's the spaces between the `]` and the `(`.

Reply:
Fixed